src/main/openapi/ds-storage-openapi_v1.yaml

```diff
@@ -264,8 +264,7 @@ paths:
         - name: mTime
           in: query
           description: >
-            Format is milliseconds with 3 added digits. It is up to the caller to keep track of 
-            mTime when batching the extracting for retrieval between separate calls.
+            Format is milliseconds with 3 added digits(see documentation in top). Start value will be excluded. It is up to the caller to keep track of mTime when batching the extracting for retrieval between separate calls, by using the last received in the next call.
```

Comment:
Please state that it's milliseconds since Epoch in each description. Could be done as simple as: _Format is milliseconds since Epoch with 3 added digits_